lib/internal/Magento/Framework/App/Config.php

```diff
@@ -1,25 +1,25 @@
 <?php
 /**
- * Application configuration object. Used to access configuration when application is initialized and installed.
- *
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```

Comment:
I think declaring strict types can broke tests. Let's try to revert it

Reply:
Thank @ihor-sviziev, I reverted it